src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -12,105 +12,325 @@ Sentry's Performance features are currently in beta. For more details about acce
 
-## Tracing & Distributed Tracing
+- What occurred for a specific error event or issue
+- The conditions that cause bottlenecks or latency issues in your application
+- The endpoints or operations that consume the most time
 
-Applications (for example, web applications) typically consist of interconnected components (also called services). For instance, let us assume a modern web application could be composed of the following components separated by the network boundaries:
+## What is Tracing?
 
-- Frontend (Single-Page Application)
-- Backend
-- Background Queue
-- Notification Job
+To begin, a note about what tracing is not: Tracing is not profiling. Though the goals of profiling and tracing overlap quite a bit, and though they are both tools which can be used to diagnose problems in your application, they differ both in terms of what they measure and how the data is recorded.
 
-Each of these components may be written in a different language on a different platform. Today, all of these components can be instrumented individually with Sentry SDKs to capture error data or crash reports whenever an event occurs in any one of them.
+A [profiler](https://en.wikipedia.org/wiki/Profiling_(computer_programming)) may measure any number of aspects of an application's operation: the number of instructions executed, the amount of memory being used by various processes, the amount of time a given function call takes, and many more. The resulting profile is a statistical summary of these measurements. 
 
-With tracing, we can follow the journey of the API endpoint requests from their source (for example, from the frontend), and instrument code paths as these requests traverse each component of the application. This journey is called a [**trace**]({%- link _documentation/performance/performance-glossary.md -%}#trace). Traces that cross between components, as in our web application example, are typically called **distributed traces**.
+A [tracing tool](https://en.wikipedia.org/wiki/Tracing_(software)), on the other hand, focuses on _what_ happened (and when), rather than how many times it happened or how long it took. The resulting trace is log of events which occurred during a program's execution, often across multiple systems. Though traces most often - or, in the case of Sentry's traces, always - include timestamps, allowing durations to be calculated, measuring performance is not their only purpose. They can also show the ways in which interconnected systems interact, and the ways in which problems in one can cause problems in another.
```

Comment:
We do actually define "distributed trace" in the first paragraph of the "Traces, Transactions, and Spans" section. My feeling here is that the less detail, the better, since this is meant to be a very high-level overview of what tracing is - really only defining it enough to show why it's not profiling. Vocabulary can come later. 🙁